src/Jackalope/Transport/DoctrineDBAL/Client.php

```diff
@@ -2517,7 +2363,10 @@ public function rollbackTransaction()
 
         try {
             $this->inTransaction = false;
-            $this->namespaces = array();
+
+            // reset namespaces
+            $this->namespaces->exchangeArray(array());
+            $this->getNamespacesObject();
```

Comment:
this line is messing up the doctrine connection with postgres. after testAddNodeInParallel the connection is already in rollback only mode... i will just copy the original namespaces for rollback. will update the PR once it runs through locally.